python_dependencies/processing/requirements.txt

```diff
@@ -1,23 +1,17 @@
-alembic>=1.9, <2
```

Comment:
this is only needed for running a migrations or creating the database, none of which are done by the processing container.